.editorconfig

```diff
@@ -0,0 +1,17 @@
+# http://editorconfig.org
```

Comment:
Added for my own sanity; I work in several projects with different defaults, and I don't lint or prettify on save.